R/point_interval.R

```diff
@@ -195,11 +194,11 @@ point_interval.default = function(.data, ..., .width = .95, .point = median, .in
       draws = data[[col_name]]
     }
 
-    result = map_dfr(.width, function(p) {
+    result = map_dfr_(.width, function(p) {
       data[[col_name]] = map_dbl_(draws, .point, na.rm = na.rm)
 
-      intervals = map(draws, .interval, .width = p, na.rm = na.rm)
-      # can't use map_dbl_ here because sometimes (e.g. with hdi) these can
+      intervals = lapply(draws, .interval, .width = p, na.rm = na.rm)
+      # can't use vapply_dbl here because sometimes (e.g. with hdi) these can
```

Comment:
Given renaming vapply_dbl -> map_dbl_ This should be:

```suggestion
      # can't use map_dbl_ here because sometimes (e.g. with hdi) these can
```